### PR TITLE
Supress 404 Error if no status was uploaded

### PR DESCRIPTION
**Pull request by timschumi**
Just don't create a file called 'generate_404' in the status branch :smirk:

### CODE REVIEW

**Comment by a7r3** [comment]
This is being fixed locally, would work on it soon

**Comment by timschumi** [comment]
How do you plan to fix it?

**Comment by a7r3** [comment]
The logic works alright if LOCAL version is less than REMOTE version.

But when it is the opposite, then the `for` loop has the limits 167 to 166, and that's feasible
So now, it tries to fetch file `167` from GitHub when it  says "404: Not Found"
and it tries to fetch file `166` from GitHub, which is successful.

So the output is...

_[Iteration 1: 167]_ 
```404: Not Found```

_[Iteration 2: 166]_
```v1.67

* Blah1
* Blah2
```

The solution for this is to set a `if` check for performing this operation ONLY when `LOCAL` version is lesser than `REMOTE` version.

I'm pushing this change with a set of derps fixed, soon

**Comment by a7r3** [comment]
Why did I close it ?! xD

**Comment by timschumi** [comment]
Y U kill mah stuff?

No really, what is the downside of this change?

**Comment by a7r3** [comment]
Nothing wrong with your method, I had these things sorted out before it was too late to continue testing and pushing it to `staging` branch.

I didn't think of the flaw which I described just now.

These changelogs would remain intact on the [status](https://github.com/a7r3/ScriBt/tree/status) branch

I liked that 404 check, but wouldn't it be like slowing down the process ?
It'd be like it's trying to fetch, fetch, and eventually says `404`. It may depend on the Internet connection. Try correcting me

File: upScriBt.sh [hunk 1]
```diff
@@ -9,7 +9,8 @@ LVER=`cat VERSION | sed -e 's/\.//'`; # Local VERSION
 echo -e "\n${INF} ${CL_WYT}Checking for Updates${NONE}\n";
 for V in `eval echo {${LVER}..$((${RVER}-1))}`; do
     MSG=`curl https://raw.githubusercontent.com/a7r3/ScriBt/status/${V} -s`;
-    [ ! -z "$MSG" ] && echo -e "${CL_WYT}${MSG}${NONE}\n";
+    PAGE404=`curl https://raw.githubusercontent.com/a7r3/ScriBt/status/generate_404 -s`;
```
Comment (a7r3) [inline upScriBt.sh]: placeholder url right ?

**Comment by a7r3** [comment]
@TimSchumi I've updated upScriBt with my changes, try adding your change, it's alright for me

**Comment by timschumi** [comment]
Maybe we could speed this up by fetching the Error Page before entering the loop.

I'll try to apply it on your changes in around half an hour

File: upScriBt.sh (hunk 1 above)
Comment (timschumi) [reply upScriBt.sh]: I just needed something for retrieving the current Error Message.

Comment (TimSchumi) [inline upScriBt.sh]: I just needed something for retrieving the current Error Message.

**Comment by timschumi** [comment]
Ready to merge (at least from my side)

**Pull request merged**